.github/workflows/docs-unstable.yml

```diff
@@ -53,8 +53,8 @@ jobs:
         -   name: Copy redirect file
             run: aws s3 cp --quiet ./docs/_build/html/_static/redirect.html s3://docs.pyscript.net/index.html
```

Comment:
We probably don't need to copy the redirect file on every push to unstable, yeah? It's just redirecting `docs/pyscript.net/index.html` to `/latest`.

Reply:
Yeah probably not, both actions have this line so I'll remove it 😄